doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -9,6 +9,19 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3357
+****************
+
+The following changes have been introduced in the v3357 of the controller:
+
+Changes
+=======
+
+* Updated the controller to be able to read RSSI from the CIS ISO channel using the ``HCI_Read_RSSI`` HCI command.
+* Fixed an issue where pins P0.28 to P0.31 cannot be allocated for controlling front-end modules (FEMs).
```

Comment:
This is mentioned in the known issues in doc/nrf/known_issues.rst, so it should be removed from there in this PR

Reply:
hmmmm, that is attached with a specific tag release, so probably the proper way is making the issue stick with (and just stop at) tag v2.4.0.

Reply:
Oh yeah, you're right. Sorry about that